### PR TITLE
#67 add implementation and test coverage of delete() method in Databa…

### PR DESCRIPTION
**Pull request by Shtramak**
connect to #67

### CODE REVIEW

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java [hunk 1]
```diff
@@ -155,6 +155,29 @@ public void insertWhenInvalidNumberOfParametersReturnsFalse() throws SQLExceptio
         assertFalse(databaseManager.insert(TEST_TABLE_NAME, tableRow));
     }
 
+    @Test
+    public void deleteWhenInvalidTableNameReturnsFalse() throws SQLException {
```
Comment (kortov) [inline src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: throws SQLException is redundant

**Review by kortov — APPROVED** [review]

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java (hunk 1 above)
Comment (Shtramak) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: You're right... and I'm confused that IDEA doesn't mark that anyhow

Comment (kortov) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: It’s higlighted with the gray color. You can inspect such errors with analyze-inspect function

Comment (Shtramak) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java]: Hm... in my version of IDEA it wasn't higlighted... Anyway thanks for your remark! )

**Review by AVM82 — APPROVED** [review]
Awesome

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java [hunk 2]
```diff
@@ -136,12 +136,22 @@ public void insertWhenValidDataReturnsTrue() throws SQLException {
     }
 
     @Test
-    public void insertWhenInvalidTableNameReturnsFalse() throws SQLException {
+    public void insertWhenTypeMismatchParameterReturnsFalse() throws SQLException {
+        databaseManager.connect(TEST_DB_NAME, DB_USER, DB_USER_PASSWORD);
```
Comment (ivshapovalov) [inline src/test/java/juja/sqlcmd/DatabaseManagerTest.java:140]: Why you use connect in every method instead of @Before?

**Review by ivshapovalov — CHANGES_REQUESTED** [review]

File: src/test/java/juja/sqlcmd/DatabaseManagerTest.java (hunk 2 above)
Comment (Shtramak) [reply src/test/java/juja/sqlcmd/DatabaseManagerTest.java:140]: There's a tests of a method connect() where shouldn't be a preliminary connection instantiating

**Review by ivshapovalov — APPROVED** [review]

**Pull request merged**